classes/cyclonedx-export.bbclass

```diff
@@ -9,76 +9,133 @@ CVE_VERSION ??= "${PV}"
 
 CYCLONEDX_EXPORT_DIR ??= "${DEPLOY_DIR}/cyclonedx-export"
 CYCLONEDX_EXPORT_SBOM ??= "${CYCLONEDX_EXPORT_DIR}/bom.json"
+CYCLONEDX_EXPORT_VEX ??= "${CYCLONEDX_EXPORT_DIR}/vex.json"
 CYCLONEDX_EXPORT_TMP ??= "${TMPDIR}/cyclonedx-export"
 CYCLONEDX_EXPORT_LOCK ??= "${CYCLONEDX_EXPORT_TMP}/bom.lock"
 
 python do_cyclonedx_init() {
     import uuid
     from datetime import datetime
 
+    timestamp = datetime.now().astimezone().isoformat()
```

Comment:
After some testing on my side, look like:
```python
from datetime import datetime, timezone
timestamp = datetime.now(timezone.utc).isoformat()
```
produce a similar result (with trailing `+00:00`) but is forced to UTC timezone instead of local timezone. Using it may be better to:
- make result more consistent
- remove unnecessary information about local time of computer producing it

Did it seem correct for you?

Reply:
Good idea!